### PR TITLE
Fix prop types

### PR DESCRIPTION
**Pull request by kesha-antonov**

### CODE REVIEW

**Comment by MarkMurphy** [comment]
This is a simple one to merge people. 😉

File: ActionSheet.android.js [hunk 1]
```diff
@@ -83,10 +83,15 @@ class ActionGroup extends React.Component {
       false
     );
 
+    let textStyle = [styles.text];
+
     for (let i = startIndex; i < startIndex + length; i++) {
       let color = '#444444';
       if (i === destructiveButtonIndex) {
         color = '#ff3b30';
+        textStyle = textStyle.concat([ this.props.textStyle, {color} ])
```
Comment (ide) [inline ActionSheet.android.js]: please fix the coding style to match the rest of this file for spacing, semis, etc. 

also please revert this change, it's a breaking change, and keep separate changes in separate PRs

Comment (xcarpentier) [reply ActionSheet.android.js]: I create new one cherry-picking only the work of @kesha-antonov on prop-types fix : https://github.com/expo/react-native-action-sheet/pull/59

**Comment by ide** [comment]
Merged the prop type PR changes in https://github.com/expo/react-native-action-sheet/pull/59

If you want the color prop changes, submit a new PR with a more flexible API.
